google/cloud/bigtable/test_proxy/BUILD.bazel

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Google LLC
```

Comment:
`google-cloud-cpp` uses CMake as the source of truth for its build scripts *and* some of the critical builds (clang-tidy for example) **require** CMake.  Please add the corresponding CMake files.

Reply:
Alternatively, work with @dbolduc to decide who gets to add them.

Reply:
I added Makefile and CMakeLists.txt. 

In my experiment, I cleared multiple blockers such that I can run command `cmake -H. -B.build -DCMAKE_TOOLCHAIN_FILE=./vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_PREFIX_PATH=~/github_proj/google-cloud-cpp/google/cloud/bigtable/test_proxy/vcpkg/packages/google-cloud-cpp_x64-linux/share`. 

The latest blocker is:
.build/CMakeFiles/CMakeTmp/CMakeLists.txt:20 (target_link_libraries):
  Target "cmTC_6649b" links to:

    gRPC::grpc++

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.